### ISSUE TITLE
The loss curve exhibits significant fluctuations, or spikes

### ISSUE THREAD

**Issue by Shexiaox** [facebookresearch/vggt#441]
<img width="1233" height="779" alt="Image" src="https://github.com/user-attachments/assets/fbed2001-4073-4308-8774-b491a24222c5" />

Is such a loss curve normal? Does it indicate data quality issues, or what steps should I take to address it?

**Comment by jytime** [comment]
Yes it looks correct to me. The peaks are usually hard random sampling examples

**Comment by sizewang23** [comment]
I would like to ask, are there any points to note during training? I am trying to freeze the AA fine-tuning and subsequent task head, but the loss still fluctuates greatly. I am using 7-scenes data and the coordinate system has been changed to w2c coordinate system. I have also tried turning off depth, reducing learning rate (LR), and decreasing depth weight. However, I still haven't found a solution. Please provide some experience and guidance
